doc/source/guide/transformer_quickstart.ipynb

```diff
@@ -343,7 +344,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df_x.replace({\"normal\": 1, \"important\": 2, \"strategic\": 3}, inplace=True)\n",
+    "df_x = df_x.replace({\"normal\": 1, \"important\": 2, \"strategic\": 3})\n",
```

Comment:
To avoid `SettingWithCopyWarning`
```python
/tmp/ipykernel_3108/3317886993.py:1: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_x.replace({"normal": 1, "important": 2, "strategic": 3}, inplace=True)
```